modules/ROOT/pages/tgw-about.adoc

```diff
@@ -0,0 +1,49 @@
+= Transit Gateway Attachments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
+
+A transit gateway is a network hub that connects your on-premises networks to your VPCs.
+The transit gateway simplifies your network connectivity by functioning
+as a cloud router, managing your VPC connections and requiring only one connection from CloudHub.
+
+Use transit gateway attachments to create a secure connection between your Virtual Private Cloud (VPC) and your private corporate networks.
+You can add multiple transit gateway attachments if required. 
+
+[NOTE]
+The number of transit gateways you can add depends on the transit gateway entitlements available to your account.
+Contact your MuleSoft account representative if you don't know how many entitlements are associated with your account. 
+
+
+== Transit Gateway Routing
+
+VPC supports static routing for transit gateway attachments.
+
+You must configure the routes (subnets) in your network that you want to be accessible through the transit gateway attachments:
+
+* On Anypoint Platform, add the AWS VPC routes to the transit gateway route table.
```

Comment:
These routes are for inbound traffic (into the VPC) from some external network. FYI this configuration doesn't always happen in AWS, and it can be pretty complex. We may have to be more vague here. The UI just says "To enable inbound traffic [to your VPC(s)] through your transit gateway, coordinate with your network administrator."

Reply:
Yeah, I fixed that elsewhere, but this escaped. Clarified.